util/tar_test.go

```diff
@@ -0,0 +1,24 @@
+package util
+
+import (
+	"testing"
+)
+
+func TestSanitizedName(t *testing.T) {
```

Comment:
Please a comment describing what this test is checking.

Reply:
Added